### ISSUE TITLE
Requirements for the Org Dashboard

### ISSUE THREAD

**Issue by Eleftherios01** [hackforla/website#7725]
### Overview
We need to outline the requirements for the Org Dashboard to ensure clear priorities, resource allocation, and effective management as we move forward with development.

### Action Items
- [x] Create Spreadsheet (version analysis feature sets)
- [ ] Go through the Roadmap and find any features/requirements it mentions, adding to a comment below
   - [ ] move the data to the sheet
- [ ] Review V1 
   - [ ] find additional features
   - [ ] indicate all the features it has and write any notes
   - [ ] Get sign-off

### Resources/Instructions
- Google Drive Folder: [Org Dashboard](https://drive.google.com/drive/folders/1MzLdRvi83kAxbMVOxeWGjnN1GH0qI-9k)
- Roadmap this dashboard: [HfLA website: Dashboard Roadmap: Org](https://docs.google.com/document/d/1HMk5vKl_MgyBTZp8mc2XDJvsKOASTfKQtE6_CplV_7k/edit?tab=t.0)

### Related Issues
- Project Board (see all issues): [HfLA Dashboards: Project Board, Org](https://github.com/orgs/hackforla/projects/88/views/4)
- https://github.com/hackforla/website/issues/7506

#### Org Dashboard Prototype Links and Notes
- v1
- [HfLA website: Org Dashboard](https://www.hackforla.org/dashboard/)
- Iframe views (for some reason, they go to different pages.  Decide which page they should go to and open issue on website team to fix links)
   -  [Web view](https://datastudio.google.com/embed/reporting/16J9XI19pQejSDNFvjl_V3ZezJ502rM_a/page/qx6RB)
   - [Mobile view](https://datastudio.google.com/embed/reporting/1xNlu1xcL5y6jh95lX3xr0zb2FCqD92Gm/page/gIfSB)
#### Requirements
- Spreadsheet [Dashboard - Org - Competitive or Comparative Analysis](https://docs.google.com/spreadsheets/d/1vqCjT_bU-7ssi0su8MhvZI0OUR8-cmtqkA6NFLWCbwc/edit?gid=471535750#gid=471535750)

**Comment by Eleftherios01** [comment]
1. Progress: I created it this issue and adjusted inserted all relevant info based on the https://github.com/hackforla/website/issues/7640. i filled the[Dashboard - Org - Competitive or Comparative Analysis Template](https://docs.google.com/spreadsheets/d/1vqCjT_bU-7ssi0su8MhvZI0OUR8-cmtqkA6NFLWCbwc/edit?gid=471535750#gid=471535750) with as many info I could extract from the  Org roadmap. The roadmap need to be reviewed by Bonnie as highlighted areas may be need editing or reajustment
3. 2. Availability: Blockers: none
4. 3. ETA: Availability:
5. 4. Pictures ETA: or links* (if necessary):
5. Pictures or links* (if necessary):

**Comment by ExperimentsInHonesty** [comment]
- [ ] Review the chart from dashboard, chart name and value (from spreadsheet) and see if they make sense to you.  If not, please add a note in column E so that we can discuss.

**Comment by ExperimentsInHonesty** [comment]
Work completed by @Eleftherios01 

- [x] add the correct urls for each v1 page
- [x] add the chart type of each chart name
- [x] add values

**Comment by Eleftherios01** [comment]
1.Progress: Reviewed the chart from dashboard, chart name and value (from spreadsheet). I added notes in column E to discuss.
2.Availability: Blockers: 
3.ETA: Availability:
4.Pictures ETA: or links* (if necessary):
5.Pictures or links* (if necessary):

**Comment by ExperimentsInHonesty** [comment]
We ran one of the chart titles through ChatGPT to see how it would reword, to make it clearer what the chart was about.  We used Eleftherios's notes 

https://chatgpt.com/share/67b2328c-02f4-8008-9ba7-dc2c24e0326b

**Comment by Eleftherios01** [comment]
1.Progress: reworded all chart names and values of Org dashboard  with the help of chatgpt. 

All Suggested Rewarding at cell F3: [Dashboard - Org - Competitive or Comparative Analysis Template](https://docs.google.com/spreadsheets/d/1vqCjT_bU-7ssi0su8MhvZI0OUR8-cmtqkA6NFLWCbwc/edit?gid=471535750#gid=471535750)

 Rewording was based on this template: [chatgpt](https://chatgpt.com/share/67b2328c-02f4-8008-9ba7-dc2c24e0326b) 

2.Availability: Blockers: nope
3.ETA: Availability:
4.Pictures ETA: or links* (if necessary):
5.Pictures or links* (if necessary):

**Comment by ExperimentsInHonesty** [comment]
Please refer to this continuation of the [conversation with Chat GPT ](https://chatgpt.com/share/67c49f3b-37c0-8008-a95b-6081514b2e5b) for formatting the values.

1. make sure each makes sense to you
2. make sure you like the format, if not, tell chatgpt how to change it.
3. reformat all the rest of the values to match the format you decide (the current example in ChatGPT are only from website general)

**Comment by ExperimentsInHonesty** [comment]
<img src="https://github.com/user-attachments/assets/f8d22d23-74a2-4810-8598-f2b5e3533973" width=300px>

New Chart recommendation for the Overall Website Score (Website: General) bottom of the page

https://www.hackforla.org/dashboard/

The current graph does not effectively represent the user satisfaction score. For better visibility, a vertical bar chart is recommended, where the x-axis represents the overall score (ranging from 1 to 10), and the record count is displayed at the top of each column.

This chart example was made at https://www.meta-chart.com/

**Comment by Eleftherios01** [comment]
1.Progress: reworded all chart names and values of Org dashboard with the help of chatgpt based on the new prompt

Additionaly, I reccomend to replace a graph chart, details in the comment above.


2.Availability: Blockers: nope
3.ETA: Availability:
4.Pictures ETA: or links* (if necessary):
5.Pictures or links* (if necessary):

**Comment by ExperimentsInHonesty** [comment]
We discussed using a stacked bar chart to combine the two charts under a user into one.  See this page [Organizational Dashboard, Website: Projects](https://lookerstudio.google.com/u/0/reporting/16J9XI19pQejSDNFvjl_V3ZezJ502rM_a/page/UPeSB) 

We would end up with two replacement charts, one for Project Leaders and one for New members, instead of the 4 we have now.

<details><summary>example</summary>

![Image](https://github.com/user-attachments/assets/6d3b3cfa-1384-4afd-abec-fdfc197c92bd)
</details>

**Comment by ExperimentsInHonesty** [comment]
Noteable events

### 2020
[Issue tab, created:2020-01-01..2020-12-31 ](https://github.com/hackforla/website/issues?q=is%3Aissue%20state%3Aclosed%20created%3A2020-01-01..2020-12-31&page=10)
- https://github.com/hackforla/website/issues/254
- https://github.com/hackforla/website/issues/273 asked for donations
- https://github.com/hackforla/website/issues/281
- https://github.com/hackforla/website/issues/282
- https://github.com/hackforla/website/issues/302 from green to pinkish grey (https://github.com/hackforla/website/issues/239)
- https://github.com/hackforla/website/issues/303
- https://github.com/hackforla/website/issues/316
- https://github.com/hackforla/website/issues/318 we started to add the names of the people who are leading projects to their project pages
- https://github.com/hackforla/website/issues/368
- https://github.com/hackforla/website/issues/384
- https://github.com/hackforla/website/issues/385
- https://github.com/hackforla/website/issues/390
- https://github.com/hackforla/website/issues/403
- https://github.com/hackforla/website/issues/404
- https://github.com/hackforla/website/issues/405
- finished with page nine, need to continue from there if we are using this methodology

The forms were set up on or around Dec 30, 2019
- https://github.com/hackforla/website/issues/242 
- https://github.com/hackforla/website/issues/316
- https://github.com/hackforla/website/issues/323
- https://github.com/hackforla/website/issues/356
- https://github.com/hackforla/website/issues/365


In order not to review all closed issues, we would need to make sure old issues have feature labels, so that we can just look at the history of one page on the website, or all pages related to meetings times, etc.

**Comment by ExperimentsInHonesty** [comment]
Will get @essencegoff working on the feature labels on closed issues.

**Comment by ExperimentsInHonesty** [comment]
@Eleftherios01 
we should have that last input received in the  [Survey Data Workbook, Google Form Links](https://docs.google.com/spreadsheets/d/1sIhdPzSpMNn7ZwGftiCbgMRtF-84ziJgmr8EjsC_yK8/edit?gid=1484716018#gid=1484716018), so that we know if the survey is no longer, or never was able to get responses.  This should enable us to fix the pipeline problem.

- [ ] please add the rest of the formulas to this page

**Comment by ExperimentsInHonesty** [comment]
We left off having reviewed the [Dashboard - Org - Competitive or Comparative Analysis Template, website: projects filter](https://docs.google.com/spreadsheets/d/1vqCjT_bU-7ssi0su8MhvZI0OUR8-cmtqkA6NFLWCbwc/edit?gid=471535750#gid=471535750&fvid=2047225764) - Eleftherios has notes about the data visualizations needing to be revised for the following:
- better charts
- data based on event (dates)
- data by year
- data by quarter over quarter
- data by last month over prior month

**Comment by ExperimentsInHonesty** [comment]
We created the following filters
- [Dashboard - Org - Competitive or Comparative Analysis Template, website: projects filter](https://docs.google.com/spreadsheets/d/1vqCjT_bU-7ssi0su8MhvZI0OUR8-cmtqkA6NFLWCbwc/edit?gid=471535750#gid=471535750&fvid=2047225764)
- [Dashboard - Org - Competitive or Comparative Analysis Template, remote onboarding filter](https://docs.google.com/spreadsheets/d/1vqCjT_bU-7ssi0su8MhvZI0OUR8-cmtqkA6NFLWCbwc/edit?gid=471535750#gid=471535750&fvid=590988561)

 - [x] make filters in the [Dashboard - Org - Competitive or Comparative Analysis Template](https://docs.google.com/spreadsheets/d/1vqCjT_bU-7ssi0su8MhvZI0OUR8-cmtqkA6NFLWCbwc) for the rest of the pages in the dashboard

**Comment by Eleftherios01** [comment]
> [@Eleftherios01](https://github.com/Eleftherios01) we should have that last input received in the [Survey Data Workbook, Google Form Links](https://docs.google.com/spreadsheets/d/1sIhdPzSpMNn7ZwGftiCbgMRtF-84ziJgmr8EjsC_yK8/edit?gid=1484716018#gid=1484716018), so that we know if the survey is no longer, or never was able to get responses. This should enable us to fix the pipeline problem.
> 
>     * [ ]  please add the rest of the formulas to this page
need access to edit the workbook, i paste the formulas here, to copy paste the formulas when i have acceess:

Remote Onboarding  3/27/2025 11:16:41
Donor/Sponsor no timestamp
Government 4/22/2024 17:15:45
Non-Profit  3/3/2025 8:47:16
Transportation & Parking 5/21/2024 16:51:20
Other 3/28/2025 6:39:08

**Comment by Eleftherios01** [comment]
1.Progress: 

1.1: I found the formulas but was unable to paste in the workbook as i have no permission. Data to be pasted are above this post.

1.2: I made filters in the [Dashboard - Org - Competitive or Comparative Analysis Template](https://docs.google.com/spreadsheets/d/1vqCjT_bU-7ssi0su8MhvZI0OUR8-cmtqkA6NFLWCbwc) for the rest of the pages in the dashboard

2.Availability: Blockers: nope
3.ETA: Availability:
4.Pictures ETA: or links* (if necessary):
5.Pictures or links* (if necessary):

**Comment by ExperimentsInHonesty** [comment]
@Eleftherios01 
The reason I have not moved this spreadsheet even though I am the owner, is that I am worried if I move it, the forms will stop writing to the sheet if they are in different locations.  I can't move the forms because they are owned by other people.  We have reached out to the other people, to get them to move the forms, but they have not responded by booking an appointment.

From Bonnie
> [@Eleftherios01](https://github.com/Eleftherios01) we should have that last input received in the [Survey Data Workbook, Google Form Links](https://docs.google.com/spreadsheets/d/1sIhdPzSpMNn7ZwGftiCbgMRtF-84ziJgmr8EjsC_yK8/edit?gid=1484716018#gid=1484716018), so that we know if the survey is no longer, or never was able to get responses. This should enable us to fix the pipeline problem.
> 
>     * [ ]  please add the rest of the formulas to this page

Eleftherios's response
>need access to edit the workbook, i paste the formulas here, to copy paste the formulas when i have acceess:

>Remote Onboarding  3/27/2025 11:16:41
>Donor/Sponsor no timestamp
>Government 4/22/2024 17:15:45
>Non-Profit  3/3/2025 8:47:16
>Transportation & Parking 5/21/2024 16:51:20
>Other 3/28/2025 6:39:08

**Comment by ExperimentsInHonesty** [comment]
@Eleftherios01 
 - [ ] Please test all the forms that are linked from the website.  just put "test" in any field that asks for a narrative response, and where it does not require validation (emails usually require validation, but you can try just putting test there too to see.
   - what we are trying to accomplish is finding out if no one has been filling out these forms because they don't work, or we just didn't have anyone who needed them, and they do work.
   - Keep track of the forms you have filled out by listing the name of the form, where you found it, and if it resulted in data being added to the spreadsheet.  You can add that data in the [survey data workbook](https://docs.google.com/spreadsheets/d/1sIhdPzSpMNn7ZwGftiCbgMRtF-84ziJgmr8EjsC_yK8/edit?gid=1185333001#gid=1185333001) in a column
- [ ] Test this form that is used during onboarding [Hack for LA new volunteer survey](https://docs.google.com/forms/d/e/1FAIpQLSdTM20e-XBD3_fmTPlYfkBKT7cv7aKdO1b2yx_OCrbHLCvLog/viewform)

**Comment by Eleftherios01** [comment]
1.Progress:

1.1: Tested all forms linked to the website. There are no issues, data appear in the spreadsheet.
These are: 
Remote Onboarding
New Member
Project Leader
Donor/Sponsor
Government
Non-Profit
Transportation & Parking
Other
https://docs.google.com/spreadsheets/d/1sIhdPzSpMNn7ZwGftiCbgMRtF-84ziJgmr8EjsC_yK8/edit?gid=1185333001#gid=1185333001

1.2: I tested https://docs.google.com/forms/d/e/1FAIpQLSdTM20e-XBD3_fmTPlYfkBKT7cv7aKdO1b2yx_OCrbHLCvLog/viewform. To ask @ExperimentsInHonesty the spreadsheet link to see results. I dont know the location of this file

2.Availability: Blockers: nope
3.ETA: Availability:
4.Pictures ETA: or links* (if necessary):
5.Pictures or links* (if necessary):

**Comment by ExperimentsInHonesty** [comment]
### To make the dashboard start working again
- [ ] We need to revise all the response sheets so that there is a response sheet from the form and a response sheet that the dashboard uses.  We did the one for Government -On the response sheet from the form, another column will need to get added that is called `Show on Dashboard`
 
### Add to requirements 
- [ ] We need a script (similar to wins), where when a new input is made into the form, it opens an issue (the [wins issues](https://github.com/hackforla/website/issues?q=state%3Aclosed%20label%3A%22new-win-submission%22)) - not sure what repo it will open the issue in.
- [ ] We need a bot that reports to the admin channel when a new issue with the appropriate label is added (this should also include the wins form inputs)
- [ ] Admin channel rep will review the input and decide if its TRUE or FALSE
- [ ] On the response sheet from the form, the Admin Channel rep will Add the TRUE or FALSE value to the column called `Show on Dashboard`

**Comment by ExperimentsInHonesty** [comment]
### Off topic - please move to Dashboard initiative issue
We need another dashboard.  Dashboard will be called Inbound Communication

**Comment by ExperimentsInHonesty** [comment]
### Screenshot of Onboarding Dashboard Page

![Image](https://github.com/user-attachments/assets/bb9771e8-b8fb-4f46-afad-243321c1d4cc)

**Comment by ExperimentsInHonesty** [comment]
https://chromewebstore.google.com/detail/gofullpage-full-page-scre/fdpohaocaechififmbbbbbknoalclacl?pli=1

**Comment by Eleftherios01** [comment]
![Image](https://github.com/user-attachments/assets/6cbf59c6-5789-43db-97c1-ba4cecde3c2d)

**Comment by Eleftherios01** [comment]
![Image](https://github.com/user-attachments/assets/82c9a313-350b-4a69-8505-d61c9f9cedd0)

**Comment by Eleftherios01** [comment]
1.Progress: I tested all survey forms to figure out where the data fall into the  [survey data workbook](https://docs.google.com/spreadsheets/d/1sIhdPzSpMNn7ZwGftiCbgMRtF-84ziJgmr8EjsC_yK8/edit?gid=1185333001#gid=1185333001) 
2.Availability: Blockers: none
3.ETA: Availability:
4.Pictures ETA: or links* (if necessary):
5.Pictures or links* (if necessary):

**Comment by ExperimentsInHonesty** [comment]
The Remote Onboarding Responses & New Member Responses  sheets have columns that should only be on the Filtered Results sheet.  But since the dashboard is using the response sheet, we will have to change it to use the Filtered Results sheet before we can delete its extra columns

### Remote Onboarding Responses Extra columns
- F
- G
- J
- K
- M
- N
- P
- Q
- S
- T
- V
- W
- Y
- Z
- AB
- AC
- AF
- AG

### New Member Responses Extra columns
- F
- G
- L
- M
- Y
- Z
- AE
- AF
- AH
- AI

**Comment by ExperimentsInHonesty** [comment]
We need to figure out how to discover which tab the data is coming from, for a particular looker chart.  Because we have two sheets with the same information, and we want to connect it to the filtered sheet.

On the [Project Leader Responses sheet](https://docs.google.com/spreadsheets/d/1sIhdPzSpMNn7ZwGftiCbgMRtF-84ziJgmr8EjsC_yK8/edit?gid=880021337#gid=880021337), column W, it has `check all that apply` and so in the [filtered results sheet](https://docs.google.com/spreadsheets/d/1sIhdPzSpMNn7ZwGftiCbgMRtF-84ziJgmr8EjsC_yK8/edit?gid=2109058260#gid=2109058260) we use two formulas to break that down into a single list, with all the instances of a check that apply.  So 

Example value on response sheet, column W:
> Agile, Kanban, Scrum, GitHub, Lean, Continuous Improvement (Kaizen), Waterfall, Other

Turns into 

Example value on filtered results sheet, column AA:
> Agile, Kanban, Scrum, GitHub, Lean, Continuous Improvement (Kaizen), Waterfall, Other

Which then turns into the list that appears in filtered results sheet column AC

> Agile
> Kanban
> Scrum
> GitHub
> Lean
> Continuous Improvement (Kaizen)
> Waterfall
> Other


---
Edit the dashboard - to get access, log into 1password, go to any of the following vaults (P:hackforla.org website, P: website devs, P: website-data science) and use [webadmin@hackforla.org](mailto:webadmin@hackforla.org) to grant edit access.
[Hack for LA | Organizational Dashboard - Desktop](https://lookerstudio.google.com/u/0/reporting/16J9XI19pQejSDNFvjl_V3ZezJ502rM_a/page/e1JSB/edit)
[Hack for LA | Organizational Dashboard - Mobile](https://lookerstudio.google.com/reporting/1xNlu1xcL5y6jh95lX3xr0zb2FCqD92Gm/page/gIfSB/edit)

**Comment by ExperimentsInHonesty** [comment]
@Eleftherios01 I met with Rabia, and she showed me how to know which tab it's pulling data from See the image

When you click on the chart (blue box on left), it shows which tab (green box on right in the dash after the name of the spreadsheet)  Example: in the image below it's pulling from Project Leader not Project Leader Responses.

![Image](https://github.com/user-attachments/assets/ee839907-8713-4295-9442-dc5502537275)

So in the [Dashboard - Org - Competitive or Comparative Analysis Template](https://docs.google.com/spreadsheets/d/1vqCjT_bU-7ssi0su8MhvZI0OUR8-cmtqkA6NFLWCbwc/edit?gid=471535750#gid=471535750&fvid=2097610759) I have added column N.  Please go to the [Org Looker Dashboard](https://lookerstudio.google.com/u/0/reporting/16J9XI19pQejSDNFvjl_V3ZezJ502rM_a/page/e1JSB/edit) click on each chart and look at the name of the tab and write that in column N of the `Dashboard - Org - Competitive or Comparative Analysis Template`

If you don't understand the direction, I will show you on Sunday, but if you do understand the direction, please get that done before next Sunday.

**Comment by ExperimentsInHonesty** [comment]
@Eleftherios01  I got an email saying there was an attempt to log in that got blocked.  Were you able to get in?

**Comment by Eleftherios01** [comment]
1.Progress: I have added to [Dashboard - Org - Competitive or Comparative Analysis Template](https://docs.google.com/spreadsheets/d/1vqCjT_bU-7ssi0su8MhvZI0OUR8-cmtqkA6NFLWCbwc/edit?gid=471535750#gid=471535750&fvid=2097610759) at the column N of each chart, the tab from which each graph pulls data from.
2.Availability: Blockers: none
3.ETA: Availability:
4.Pictures ETA: or links* (if necessary):
5.Pictures or links* (if necessary):

**Comment by ExperimentsInHonesty** [comment]
- [x] Confirm that each Data sheet (that matches the response sheet) has a TRUE at the end of its formula, and the column its referencing is `Show on Dashboard` on the corresponding Response sheet.
   - [x] Remote Onboarding
   - [x] New Member
   - [x] Project Leader
   - [x] Donor/Sponsor
   - [x] Government
   - [x] Non-Profit
   - [x] Transportation & Parking
   - [x] Other
- [ ] Mark with a x each response sheet marked true or false
   - [ ] Remote Onboarding
   - [x] New Member
   - [x] Project Leader
   - [x] Donor/Sponsor
   - [x] Government
   - [x] Non-Profit
   - [x] Transportation & Parking
   - [x] Other

**Comment by ExperimentsInHonesty** [comment]
It turns out there are two onboarding forms
1. [the old one (run out of the website dashboard drive) Remote Onboarding Survey](https://docs.google.com/forms/d/e/1FAIpQLScXnJSyCXgO_RCAuCyOkG4sqGILpAepTlJ0HOaK4H_ccEVmNw/viewform)
2. [the new one (run by CTJ) Hack for LA new volunteer survey](https://docs.google.com/forms/d/e/1FAIpQLSdTM20e-XBD3_fmTPlYfkBKT7cv7aKdO1b2yx_OCrbHLCvLog/viewform)

- [x] Create a new spreadsheet in our folder, and create three columns [onboarding form question comparison](https://docs.google.com/spreadsheets/d/1GxPglqPRW2GRVJUanfkVBsik1SspIimlF2NGRNn9vmo)
   - Questions (text)
   - OLD (check boxes)
   - NEW (check boxes)
   - question ID that it is similar to
- [x] Add each question to a row
- [x] add reference to the other question id if questions are the same or similar (example question)

**Comment by ExperimentsInHonesty** [comment]
We have to find out where people are filling out the old new member form.  Because it was determined by us at this meeting that none of the people who filled it out, had attended the second half of onboarding.

when the above is done, Bonnie and Eleftherios will decide how to reconcile the two forms, so that we have form, one data set.

**Comment by ExperimentsInHonesty** [comment]
Create a new issue in this repo https://github.com/hackforla/data-science/issues


```
### Overview
We need to fix a broken data pipeline so that our dashboard will display properly


#### Details
Google Sheets runs our Looker Dashboard.  It has become disconnected

### Action Items
- [ ] get access to dashboard copy
- [ ] get access to sheet via 
   - [ ] get added to drive
   - [ ] add yourself to roster
- [ ] get calendar invite for Sunday at 9am PST HfLA Dashboard meeting

### Resources/Instructions
- Google  Drive: H[fLA: Dashboards](https://drive.google.com/drive/folders/0AAinpuClsquoUk9PVA)
   - Website team roster
   - URL of Google Sheet Looker is using: [Survey Data Workbook, Remote Onboarding]

#### Original
- Looker Dashboard page EDIT, [Remote Onboarding]( https://lookerstudio.google.com/u/0/reporting/16J9XI19pQejSDNFvjl_V3ZezJ502rM_a/page/qx6RB/edit)
- Looker Dashboard page VIEW, [Remote Onboarding](https://lookerstudio.google.com/embed/reporting/16J9XI19pQejSDNFvjl_V3ZezJ502rM_a/page/qx6RB)

#### Copy
- Looker Dashboard page EDIT, [Remote Onboarding]( https://lookerstudio.google.com/u/0/reporting/e05c8afa-2b84-4286-8f57-4c8b6b40fd82/page/qx6RB/edit
- Looker Dashboard page VIEW, [Remote Onboarding](https://lookerstudio.google.com/u/0/reporting/e05c8afa-2b84-4286-8f57-4c8b6b40fd82/page/qx6RB)




```
